chemprop/uncertainty/evaluator.py

```diff
@@ -68,8 +82,12 @@ def evaluate(self, preds: Tensor, uncs: Tensor, targets: Tensor, mask: Tensor) -
 @UncertaintyEvaluatorRegistry.register("spearman")
 class SpearmanEvaluator(UncertaintyEvaluator):
     def evaluate(self, preds: Tensor, uncs: Tensor, targets: Tensor, mask: Tensor) -> Tensor:
-        ...
-        return
+        masked_preds = preds * mask
+        masked_targets = targets * mask
+        masked_uncs = uncs * mask
+        masked_errs = (masked_preds - masked_targets).abs()
+        spearman = SpearmanCorrCoef(num_outputs=masked_targets.shape[1])
+        return spearman(masked_uncs, masked_errs).unsqueeze(0)
```

Comment:
I suspect that `.unsqueeze(0)` only needs for `num_outputs=1`. Could you double check this?

Reply:
Yes, I believe you're right. I'll fix this while addressing the masking issue.